api/api-private-tenant-permissions.go

```diff
@@ -56,7 +56,7 @@ func (s *privateServer) TenantPermissionAdd(ctx context.Context, in *pb.TenantPe
 	}
 	appCtx.Tenant = &tenant
 
-	if _, err := cluster.AddPermission(appCtx, in.Name, in.Description, effect, in.Resources, in.Actions); err != nil {
+	if _, err := cluster.AddPermissionToDB(appCtx, in.Name, in.Description, effect, in.Resources, in.Actions); err != nil {
```

Comment:
technically this adds the permission to the DB and also commits the change to MinIO

Reply:
ooki